includes/admin/class-sensei-extensions.php

```diff
@@ -137,6 +138,34 @@ function( $extension ) use ( $installed_plugins ) {
 		return $extensions;
 	}
 
+	/**
+	 * Get extensions page layout.
+	 *
+	 * @since 3.11.0
+	 *
+	 * @return array
+	 */
+	public function get_layout() {
+		$transient_key    = implode( '_', [ 'sensei_extensions_layout', get_locale() ] );
+		$extension_layout = get_transient( $transient_key );
+		if ( false === $extension_layout ) {
+			$raw_layout = wp_safe_remote_get(
+				add_query_arg(
+					[ 'lang' => get_locale() ],
```

Comment:
Admin pages use the user's language, so this should be `determine_locale()` — that will pick up the same locale the rest of the page is served with. (It will return `get_user_locale()` in normal circumstances).

(Also in the other places it's used)

Reply:
Interesting! I fixed it for these cases related to this API. https://github.com/Automattic/sensei/pull/4204/commits/01cad55dc905101874e8af3d96e5d0f5156db580
I still need to fix some conflicts in this PR.